<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSettingView</class>
 <widget class="QWidget" name="VideoSettingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider#videoProgressBar {
}
QSlider#videoProgressBar::groove {
	border: none;
    height: 4px; 
    background: #A0A0A0A0; 
    margin: 1px -7px;
}

QSlider#videoProgressBar::handle	{
    background: transparent;
    border: none;
    width: 18px;
    margin: -5px 0;
    border-radius: 3px;
}
QSlider#videoProgressBar::handle:hover	{
    background: #66B2FF;
    border: none;
    width: 14px;
    margin: -5px 5px;
    border-radius: 7px;
}

QSlider#videoProgressBar::sub-page {
    border: none;
    height: 4px; 
	background: #66B2FF;
    margin: 1px 0px;
}

QToolButton {
	background: transparent;
	color: white;
	font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="infoDisplayContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="videoProgressBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="settingBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="playerButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/play_white.png</normaloff>:/icons/play_white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BackwardButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/backward_track_white.png</normaloff>:/icons/backward_track_white.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="FowardButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/forward_track_white.png</normaloff>:/icons/forward_track_white.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="videoTimeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>00:00/00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="voiceSettingButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/volume_3_white.png</normaloff>:/icons/volume_3_white.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clarityButton">
        <property name="text">
         <string>1080p</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sourceButton">
        <property name="text">
         <string>Ê∫ê</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settingButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/settings_white.png</normaloff>:/icons/settings_white.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="showFullScreenButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/icons/showFullScreen_white.png</normaloff>:/icons/showFullScreen_white.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
